coverage/clover.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<coverage generated="1498995873564" clover="3.2.0">
```

Comment:
previously i had done it, but after forgot. Now, i have done this + __snapshots__ also added to gitignore